libs/langchain/langchain/chains/synthetic_data/prompts.py

```diff
@@ -0,0 +1,10 @@
+from langchain.prompts.prompt import PromptTemplate
+
+sentence_template = """Given the following fields, create a sentence about them. Make it detailed and interesting.
+
+Fields:
+{fields}
+Sentence:
```

Comment:
Apparently newline at the end of the prompt isn't a good idea - GPT probably won't care, but open-source models (GPT-J) tends to screw up the probabilities :smile: 

I encountered this while testing GPT-J:
![image](https://github.com/deepsense-ai/langchain/assets/142261444/83e3bbaf-8a4c-4065-802b-8f3840600651)

In LangChain, from what I've looked at, they don't give that newline character either